packages/core/src/components/Icon/__stories__/Icon.stories.tsx

```diff
@@ -84,7 +86,42 @@ export const Color = {
 };
 
 export const IconsListStory = {
-  render: IconsList.bind({}),
-  name: "Icons List",
```

Comment:
maybe leave this `name` attribute (or just change the `const` name to IconsList`), so it won't be shown as "Icons List Story" in the docs page